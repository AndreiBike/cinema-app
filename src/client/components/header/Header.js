import React from 'react';
import Search from '@root/client/components/header/search/Search'
import './Header.module.css';
// functional component
export class Header extends React.Component {

  constructor(props){
    super(props);
  }

  render() {
    const {
      headerLabels: {
        netflixRoulette,
        findYourMovie,
      }
    } = this.props;
    return (
      <div className="header">
        <p className="header-netflixroulette">
          <span className="header-netflixroulette-bold">{netflixRoulette[0]}</span>{netflixRoulette[1]}
        </p>
        <div className="header-find-your-movie">
          {findYourMovie}
        </div>
        <Search />
      </div>
    )
  }
}

