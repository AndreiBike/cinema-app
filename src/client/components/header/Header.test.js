import React from 'react';
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from './Header'

configure({adapter: new Adapter()});

describe("<Header /> component testing", ()=> {
  let component, headerLabelsTest;
  headerLabelsTest = {
    netflixRoulette: ["NETFLIX", "ROULETTE"],
    findYourMovie: "FIND YOUR MOVIE",
  };
  component = shallow(<Header headerLabels = {headerLabelsTest} />);
  it("test #1 snapshot testing", ()=>{
    expect(component).toMatchSnapshot();
  })
})