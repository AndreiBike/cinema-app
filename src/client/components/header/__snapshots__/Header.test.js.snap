// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component testing test #1 snapshot testing 1`] = `ShallowWrapper {}`;
